.github/workflows/ci.yml

```diff
@@ -49,6 +49,16 @@ jobs:
       - name: Test examples
         run: make test-examples
 
+  parallel-tests:
```

Comment:
For uniformity, this should be called either `parallel-examples` or just `parallel`  (the test part is implied).